terraform/ec2/amis.tf

```diff
@@ -36,7 +36,7 @@ variable "ami_family" {
       otconfig_destination     = "/tmp/ot-default.yml"
       download_command_pattern = "curl %s --output aws-otel-collector.rpm"
       install_command          = "sudo rpm -Uvh aws-otel-collector.rpm"
-      start_command            = "sudo /opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl -c \"$(echo -n 'CONFIGURATION_URI_PLACEHOLDER' | base64 -d)\" -a start"
+      start_command            = "sudo /opt/aws/aws-otel-collector/bin/aws-otel-collector-ctl -c \"$(echo -n 'CONFIGURATION_URI_PLACEHOLDER' | base64 -d)\" -f $FEATUREGATE_PLACEHOLDER -a start"
```

Comment:
not familiar with terraform, but `replace` seems to just replace the string as is.

do we need the `$` before `FEATUREGATE_PLACEHOLDER`?

Reply:
Terraform is replaces it as a literal string, hence it wouldn't be required, committed the changes.